src/framework/genesis_framework.gemspec

```diff
@@ -8,11 +8,12 @@ Gem::Specification.new do |gem|
   gem.email = 'opensourcesoftware@tumblr.com'
 
   gem.date = '2016-01-13'
-  gem.version = '0.6.8'
+  gem.version = '0.6.10'
   gem.add_dependency('genesis_promptcli', '~> 0.2', '>= 0.2.0')
   gem.add_dependency('genesis_retryingfetcher', '~> 0.4', '>= 0.4.0')
   gem.add_dependency('collins_client', '~> 0.2', '>= 0.2.0')
   gem.add_dependency('facter', '~> 2.0', '>= 2.0.0')
+  gem.add_dependency('activesupport', '5.1.1')
```

Comment:
'activesupport' is a rather large gem with a bunch of dependencies so I'm hesitant to add it if we only need a very small part.  Even 'active_support/hash_with_indifferent_access' seems pretty heavy weight for our purposes.